main.go

```diff
@@ -63,7 +62,7 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:             scheme,
-		MetricsBindAddress: metricsAddr,
+		MetricsBindAddress: constants.GetMetricBind(),
```

Comment:
what's the reason for this change?

Reply:
I guess it's to make K8up "configurable with EnvVars only"? Best option IMO would be to use Viper or even better Koanf to make K8up configurable with both flags and Envvars, but that introduces a new library.

Reply:
In master it was made setting a default value using viper. Should I use Koanf package instead and take the value from the EnvVars or set a default (if the value is not present in EnvVars)? With flags you meant to have a default, right?

Reply:
It makes sense to have this configurable via EnvVars. I don't know Koanf, but jusdging from their Github page it looks like a lighter Viper alternative. So that's fine by me.

Reply:
ok, I'll use Koanf, thanks

Reply:
You can have a look into Seiso where we use Koanf too: https://github.com/appuio/seiso/blob/master/cmd/root.go#L49

Reply:
@ccremer should I also follow the same guidelines for the env vars loading or it’s not something generic you use in all projects? https://github.com/appuio/seiso/blob/fddf46ce9081c232cb98084f22abe254fd889165/cmd/root.go#L101